Tests/WdUnitTestCase.php

```diff
@@ -171,4 +171,9 @@ protected function isValidDbDataSet(array $dbDataSet): bool
     {
         return !empty($dbDataSet['table']) && !empty($dbDataSet['columns']) && !empty($dbDataSet['rows']);
     }
+
+    protected function assertEqualsIgnoreOrder($mExpected, $mActual, $sMessage = '')
```

Comment:
In PHPUnit 9+, this is called `assertEqualsCanonicalizing`, see https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Assert.php#L611

Should we call it the same?

Reply:
makes sense!